stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/ChatEventHandler.kt

```diff
@@ -64,6 +66,18 @@ public abstract class BaseChatEventHandler : ChatEventHandler {
         filter: FilterObject,
     ): EventHandlingResult
 
+    /** Handles [MemberAddedEvent] event. It runs in background. */
+    public abstract fun handleMemberAddedEvent(
+        event: MemberAddedEvent,
+        filter: FilterObject,
+    ): EventHandlingResult
+
+    /** Handles [MemberRemovedEvent] event. It runs in background. */
+    public abstract fun handleMemberRemovedEvent(
```

Comment:
We can't make it abstract. Because we break current implementation of our customers. We can provide default implementation with `SKIP` result